astro-migration-provider/astronomer/aeroscope/operators.py

```diff
@@ -0,0 +1,88 @@
+import base64
```

Comment:
`rm`

Reply:
@fritz-astronomer this is used on line 45 of the same file?

Reply:
sorry - I meant the whole file. If we are managing it in https://github.com/astronomer/telescope (as per conversation earlier today)